selection/select_ligand_and_surroundings/src/index.ts

```diff
@@ -0,0 +1,30 @@
+import { StructureSelectionQueries, StructureSelectionQuery } from "molstar/lib/mol-plugin-state/helpers/structure-selection-query";
+import { MolScriptBuilder as MS } from 'molstar/lib/mol-script/language/builder';
+import { createRootViewer } from "./common/init";
+
+async function init() {
+    // Create viewer
+    const plugin = await createRootViewer();
+    
+    // Download PDB
+    const fileData = await plugin.builders.data.download(
+        { url: "https://models.rcsb.org/4hhb.bcif", isBinary: true }
+    );
+
+    // Load PDB and create representation
+    const trajectory = await plugin.builders.structure.parseTrajectory(fileData, "mmcif");
+    await plugin.builders.structure.hierarchy.applyPreset(trajectory, "default");
+
+    // Query all ligands using prebuilt query
+    const ligandExp = StructureSelectionQueries.ligand.expression;
```

Comment:
would be useful to explain what we would do if there is more than one ligand, perhaps we can use an example that has multiple and just pick one?

Reply:
It all depends on how you decide which ligand is the one you want. If you already know the chain of your ligand, you would use a `'chain-test': MS.core.rel.eq([MS.ammp('label_asym_id'), 'A'])`. If you know the residue, you would use a residue test.

Here I think we should just show off selecting all ligands, because once it goes down to selecting a single ligand... its really case by case